### DISCUSSION TITLE
NBC 30 Connecticut [New Britain/Hartford/New Haven, Connecticut]

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#366] [Show and tell]
### Channel ID

WVITDT1.us

### Stream URL

https://nbculocallive.akamaized.net/hls/live/2037086/hartford/stream1/master.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://nbculocallive.akamaized.net/hls/live/2037086/hartford/stream1/master.m3u8 (offline)
